packages/calcite-design-tokens/src/component/[template-comp-name].json

```diff
@@ -0,0 +1,177 @@
+{
+  "[comp-name]": {
```

Comment:
Can we avoid abbreviations, e.g. `component-name` instead of `comp-name`. Applies to anywhere else abbreviations are used.